### PR TITLE
Update README.md

### PR DESCRIPTION
**Pull request by Strflightmight09**

### CODE REVIEW

**Review by Nummun14 — APPROVED** [review]

**Review by levyishai — CHANGES_REQUESTED** [review]
Wrong link, should link to the TRIGON drive.

**Review by Nummun14 — APPROVED** [review]

File: README.md [hunk 1]
```diff
@@ -1,16 +1,16 @@
 # RobotCodeOffSeason2024
 
-TRIGON 5990's robot code for the 2024 off-season robot, "PLACEHOLDER".
+TRIGON 5990's robot code for the 2024 off-season robot, "Killshon".
 
 # External Programs
 
 All the "Advantage Scope" related files (CAD glTF files, layouts, etc.) can be
-found [here](https://www.youtube.com/watch?v=dQw4w9WgXcQ)   
+found [here](https://drive.google.com/drive/folders/1mMYOdDzlApMwTz2Wo5Z5GBv4ffCnPAjh?usp=drive_link)
```
Comment (levyishai) [inline README.md]: Layout name is still tragedy.
And I also just noticed there's no comma in the end of the sentence here.

**Review by levyishai — CHANGES_REQUESTED** [review]

File: README.md [hunk 2]
```diff
@@ -1,16 +1,16 @@
 # RobotCodeOffSeason2024
 
-TRIGON 5990's robot code for the 2024 off-season robot, "PLACEHOLDER".
+TRIGON 5990's robot code for the 2024 off-season robot, "Killshon".
 
 # External Programs
 
 All the "Advantage Scope" related files (CAD glTF files, layouts, etc.) can be
-found [here](https://www.youtube.com/watch?v=dQw4w9WgXcQ)   
+found [here](https://drive.google.com/drive/folders/1mMYOdDzlApMwTz2Wo5Z5GBv4ffCnPAjh?usp=drive_link).
```
Comment (Nummun14) [inline README.md:8]: image
it still says Tragedy in the layout :)

**Review by Nummun14 — CHANGES_REQUESTED** [review]

File: README.md (hunk 2 above)
Comment (Nummun14) [inline README.md:8]: The name is still tragedy I'm the config file :)

Comment (Nummun14) [inline README.md:8]: The name is still tragedy in the config file :)

**Review by Nummun14 — CHANGES_REQUESTED** [review]

**Review by Nummun14 — APPROVED** [review]

**Review by levyishai — APPROVED** [review]

**Pull request merged**